src/common/hooks/useDefaultTrendsPeriod.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * On mobile screens, we default the Trends view to show past-180-days
+ */
+
+import { useEffect, useState } from 'react';
+import useBreakpoint from './useBreakpoint';
+import { Period } from 'components/Explore/utils';
+
+export default function useDefaultTrendsPeriod(): Period {
+  const isMobile = useBreakpoint(800);
+  const [defaultTimePeriod, setDefaultTimePeriod] = useState<Period>(
+    Period.ALL,
+  );
+
+  useEffect(() => {
+    if (isMobile) {
+      setDefaultTimePeriod(Period.DAYS_180);
+    } else {
+      setDefaultTimePeriod(Period.ALL);
+    }
+  }, [isMobile]);
+  return defaultTimePeriod;
```

Comment:
I'm not sure you need the `defaultTimePeriod` state value and this `useEffect()` call.

I think you could just do:
```
export default function useDefaultTrendsPeriod(): Period {
  const isMobile = useBreakpoint(800);
  return isMobile ? Period.DAYS_180 : Period.ALL;
}
```

And FWIW at that point there's not that much value to having this `useDefaultTrendsPeriod()` hook that's only used in one place.  I'd probably just use the `useBreakpoint()` hook directly from the explore component. 🤷